jaeger_client/tracer.py

```diff
@@ -230,6 +235,11 @@ def report_span(self, span):
     def random_id(self):
         return self.random.getrandbits(constants.MAX_ID_BITS)
 
+    def is_allowed(self, *args, **kwargs):
```

Comment:
this function name makes no sense in this context

Reply:
True. I will use the name found in the Go client (`isDebugAllowed`), which is more clear. Will fix this.